XestiMonitors.xcodeproj/project.pbxproj

```diff
@@ -123,6 +123,7 @@
 		BDD79C39EC160D102A6C598C591B866C /* DocumentStateMonitor.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4753DA906CAAC43AA95E1CF6AC622AA /* DocumentStateMonitor.swift */; };
 		CF7869DFFFA2FF0445DC4DA6B6D33566 /* DocumentStateMonitor.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4753DA906CAAC43AA95E1CF6AC622AA /* DocumentStateMonitor.swift */; };
 		CFB7376E05C361DEEB6F1F24D5880AD2 /* DocumentStateMonitorTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8650D5F1A9FF403E7779526361A70B01 /* DocumentStateMonitorTests.swift */; };
+		B9E9389C82EFDCDD50CA99CE41B17602 /* Examples in Resources */ = {isa = PBXBuildFile; fileRef = 5ACFE6A7C6A7A80209B33B4555252CDE /* Examples */; };
```

Comment:
It looks like you (accidentally?) added the `Examples` folder to this project. That does not belong in here.

Reply:
I have removed it.